### PR TITLE
Ab feature/add display name

### PR DESCRIPTION
**Pull request by boltronics**
Work on addressing https://github.com/santouras/foosball/issues/24

I think I'm going to need a hand with rspec test cases before this can get merged in. There's also the question of if we want to replace username in the model with email.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -7,4 +7,16 @@ class User < ActiveRecord::Base
   def games
     Game.where('user1_id = :id OR user2_id = :id', id: id)
   end
+
+  def display_name
```
Comment (santouras) [inline app/models/user.rb]: this could be shortened to
```ruby
def display_name
  nickname || full_name
end
```

**Comment by santouras** [comment]
to get the tests password, will need to update the `user` factory that sits in `spec/factories/users.rb` to have `username` instead of `display_name`

File: app/models/user.rb (hunk 1 above)
Comment (boltronics) [inline app/models/user.rb]: Thanks. I knew there was a shorter way somehow. :)

**Comment by santouras** [comment]
Moved to #41

**Pull request closed without merging**